proposals/0118-partitioned-epoch-reward-distribution.md

```diff
@@ -0,0 +1,274 @@
+---
```

Comment:
Hmm, that authors list is particularly not good. I'll see if I can force it to be a list within the table display

Reply:
yeah and the zero-prefixed numbers being interpreted as octal is... surprising

Reply:
> yeah and the zero-prefixed numbers being interpreted as octal is... surprising

Ah, that one I can fix by using single quotes. But agreed.

Reply:
what does this look like?
```
accounts: Haoran Yi (Anza)<br>Justin Starry (Anza)<br>Tyera Eulberg (Anza)
```

Reply:
:sob: 

how about
```
accounts: >-
Haoran Yi (Anza)
Justin Starry (Anza)
Tyera Eulberg (Anza)
```

Reply:
Looks like EIP just uses a [comma separated list](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md). I'd be fine with that since it has a precedent. I'll update the other multi-author SIMDs later if we go with that.

Reply:
> how about

It looks like this 😂 
<img width="1067" alt="Screenshot 2024-02-20 at 10 47 55 PM" src="https://github.com/solana-foundation/solana-improvement-documents/assets/3030561/fc523497-51a9-4d7a-82c8-fb9156dc74f5">

I'll push comma-separated list